README.md

```diff
@@ -0,0 +1,16 @@
+# Satellite Ansible Roles
+
+Ansible Roles to setup / configure Red Hat Satellite. The roles will be created using [foreman-ansible-modules](https://github.com/theforeman/foreman-ansible-modules) or FAM.
+
+Note: The roles here are not officially supported by Red Hat.
+
+## Rules for contributing Roles to this project
+
+* The use cases supported by the roles should be beneficial for many users.
+* Input variables externalized in `vars` folder for easy reuse.
+* The roles should be compatible with the currently (upstream) supported Ansible versions, 2.7+ at the time of writing.
+* The roles should be compatible with any Python supported by Ansible upstream (2.7, 3.5, 3.6, 3.7 at the time of writing), however, roles usually don’t require any special Python features (in contrast to modules).
```

Comment:
Can we add any simple CI to vet these rules at a basic level? @evgeni

Reply:
https://github.com/RedHatSatellite/satellite-ansible-roles/issues/2

Reply:
probably should be done via molecule